src/modules/shared/forms/components/input.component.ts

```diff
@@ -87,4 +89,12 @@ export class FormInputComponent extends ControlValueAccessorConnector implements
     this.fieldChangeSubscription.unsubscribe();
   }
 

```

Comment:
This is too much responsibility for this component. I would like to discuss another approach...

Reply:
Reverted the changes as suggested